partneredyoutube script batch.BAT

```diff
@@ -0,0 +1,6 @@
+@echo off
+Title Running PartneredYoutube Verification Script v1
+dir "C:\Users\Kyle Shurtleff\Documents\GitHub\partnered-youtube-reddit"
```

Comment:
the directory should be a relative URL rather than absolute.

example: `.\partnered-youtube-reddit`. Will need to test though (i'm not too confident about windows)